var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -631,8 +631,8 @@ def define_tpl(trilinos_name, spack_name, have_dep):
                 ]),
                 define('ParMETIS_LIBRARY_NAMES', ['parmetis', 'metis']),
                 define('TPL_ParMETIS_INCLUDE_DIRS', [
-                    spec['parmetis'].prefix.include,
-                    spec['metis'].prefix.include
+                    spec['parmetis'].headers.directories[0],
+                    spec['metis'].headers.directories[0]
```

Comment:
Same:
```suggestion
                    spec['parmetis'].headers.directories,
                    spec['metis'].headers.directories
```

Reply:
Picking up the conversation from above, the edits you suggest do _not_ work here.  Perhaps the `define` function is not capable of dealing with two lists separated by a comma but the end result is that the variable ends up with a python list (with '[' ']' and everything), likewise yielding unusable CMake artifacts.

Reply:
Oh my bad, I didn't notice these two are already inside a list. Try:
```python
                define('TPL_ParMETIS_INCLUDE_DIRS',
                        spec['parmetis'].headers.directories + spec['metis'].headers.directories[0]
                ),
```
(formatting might not be right)

Reply:
Yep, that's it - pushing a change like that shortly (once I test it) ...